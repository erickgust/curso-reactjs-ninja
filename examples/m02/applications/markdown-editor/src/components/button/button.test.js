import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from './index'

it('Default Button should match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null}>
      Default
    </Button>
  ).toJSON()

  expect(tree).toMatchSnapshot()
})

it('Success Button should match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null} kind='success'>
      Success
    </Button>
  ).toJSON()

  expect(tree).toMatchSnapshot()
})

it('Danger Button should match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null} kind='danger'>
      Danger
    </Button>
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
