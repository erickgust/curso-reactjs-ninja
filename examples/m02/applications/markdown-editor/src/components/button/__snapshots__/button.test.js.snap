// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Danger Button should match snapshot 1`] = `
<button
  className="main-button -danger"
  onClick={[Function]}
>
  Danger
</button>
`;

exports[`Default Button should match snapshot 1`] = `
<button
  className="main-button "
  onClick={[Function]}
>
  Default
</button>
`;

exports[`Success Button should match snapshot 1`] = `
<button
  className="main-button -success"
  onClick={[Function]}
>
  Success
</button>
`;
